### DISCUSSION TITLE
Let's Review!

### DISCUSSION THREAD

**Discussion by dmarro89** [dmarro89/go-redis-hashtable#2] [General]
Please, feel free to review the code and add any comments positive or negative if you want.
Any feedback is really appreciated.

**Comment by dmarro89** [comment]
Please @vdmitriyev could you take a look to this ?
Is the hashtable implementation used by DareDB

**Comment by vdmitriyev** [comment]
@dmarro89 thank for information. I will give it a try.

**Reply by vdmitriyev** [reply]
@dmarro89 small update from me: my review is still in progress

**Reply by dmarro89** [reply]
> @dmarro89 small update from me: my review is still in progress

Really appreciate it.
Thanks
